CHANGELOG.md

```diff
@@ -1,25 +1,70 @@
-# NEXT MAJOR RELEASE
```

Comment:
It doesn't look like this conflict was resolved correctly. Where did all the next-major changes go?